app/uk/gov/hmrc/cgtpropertydisposalsstubs/controllers/FinancialDataController.scala

```diff
@@ -33,6 +33,11 @@ class FinancialDataController @Inject() (cc: ControllerComponents) extends Backe
     dateFrom: String,
     dateTo: String
   ): Action[AnyContent] = Action { _ =>
+    logger.info(s"""
+        idType = $idType, idNumber = $idNumber,
```

Comment:
it'd be handy to have an indication about what endpoint is being called in the log message, e.g. `"Get financial data called with parameters: idType = $idType..."`